packages/manager/modules/billing/src/components/service-status/service-status.html

```diff
@@ -0,0 +1,19 @@
+<div>
+    <span
+        class="oui-badge oui-badge_error"
+        data-ng-if="$ctrl.service.hasDebt()"
+        data-translate="manager_billing_service_status_pending_debt"
+    >
+    </span>
+    <span data-ng-if="$ctrl.service.isOneShot()">-</span>
+    <span
+        class="oui-badge"
+        data-ng-class="{
+        'oui-badge_error': $ctrl.SERVICE_STATES.error.includes($ctrl.service.getRenew()),
```

Comment:
**question:** Using `.includes` cannot be polyfilled from template I think. Does it was already the case?

Reply:
@antleblanc this was just extracted so it could be reused as a component. Nothing was changed in the logic apart maybe the translations prefix